bin/mailing-common/mailing-list-utils.rb

```diff
@@ -74,3 +74,27 @@ def query_subscribed_contacts(params)
     ALL.merge! results
   end
 end
+
+EMAIL = 'Email Address'
+NAME = 'Name'
+COUNTRY_CODE = 'CC'
+COUNTRY_CODE_US = 'US'
+
+def include_mailchimp_engineers(results)
```

Comment:
ALL is kind of weird though.. but this is going away soon when you rewrite the mail system, right? :)

are the numbers the same as before?